### ISSUE TITLE
it's possible to remove necessary css and break above-the-fold

### ISSUE THREAD

**Issue by leeoniya** [ABVanton200/critical-css-parser#1]
@ABVanton200 

```css
#card {
  position: absolute;
  top: 1000%;
}
```

this will probably remove this `#card` element, then drop its CSS and on the real page it will now (wrongly) appear above the fold.

i can't think of a solution to this. but you may want to mention that stuff like this can happen if elements are moved below the fold by css only.

**Comment by ABVanton200** [comment]
Thanks, @leeoniya, for the report!

This is an interesting type of problems that may occur. I will try to find solutions to this issue.

But now, if such a problem has been discovered, the developer can easily add this selector to the white list 👍 

For example:
```js
const result = await criticalCSSParser({
    type: 'HTML',
    html,
    css,
    minify: true,
    whitelist: /#card/
});
```

**Comment by ABVanton200** [comment]
Fixed in 1.1.0
